articles/storage/blobs/access-tiers-overview.md

```diff
@@ -84,6 +84,9 @@ To change the redundancy configuration for a storage account that contains blobs
 
 Migrating a storage account from LRS to GRS is supported as long as no blobs were moved to the archive tier while the account was configured for LRS. An account can be moved back to GRS if the update is performed less than 30 days from the time the account became LRS, and no blobs were moved to the archive tier while the account was set to LRS.
 
+> [!NOTE]
+>Beginning with version 2021-02-12, you can copy an archived blob to an online tier in a different storage account, as long as the destination account is in the same region as the source account
```

Comment:
Thank you for taking the time @animeshdas11! This note relates to the process of rehydration. We cover that subject in detail in this article [Blob rehydration from the Archive tier](https://learn.microsoft.com/en-us/azure/storage/blobs/archive-rehydrate-overview#copy-an-archived-blob-to-an-online-tier). In fact, we have something very close to this exact note in the "Copy an archived blob to an online tier" section of that article. I would avoid duplicating even one note from that article here as it opens up the flood gate of adding more duplicative notes. This breaks our intent to drive eyeballs to the page about this subject. If you are concerned about discoverability, I would recommend that we add a link to this article instead ([Blob rehydration from the Archive tier](https://learn.microsoft.com/en-us/azure/storage/blobs/archive-rehydrate-overview#copy-an-archived-blob-to-an-online-tier) where readers can get the entire body of guidance on the subject and not just one note.